packages/layout/src/components/cards/cards.module.scss

```diff
@@ -9,33 +8,81 @@
 	border: 1px solid $colors-neutral-4;
 	box-shadow: 0 2px 3px rgba(0, 0, 0, 0.2);
 	margin-bottom: $space-6;
-}
 
-.cardToolbar {
-	display: flex;
-	flex-direction: row;
-	flex: 0 0 auto;
-	min-height: 100px;
-	padding: 40px 40px 0 40px;
-	border-left: 5px solid;
-	border-color: $colors-danger-2;
-}
+	.cardToolbar {
+		display: flex;
+		flex-direction: row;
+		flex: 0 0 auto;
+		min-height: 100px;
+		padding: 40px 40px 20px 40px;
+		border-left: 5px solid;
+		border-color: $colors-danger-2;
 
-.complete {
-	border-color: $colors-success-1;
-}
+		.verticalHr {
```

Comment:
.verticalHorizontalRule? 

.divider might be a better name. It describes the purpose, rather than the current implementation.

Reply:
renamed to `.divider`